third_party/amd/backend/compiler.py

```diff
@@ -204,7 +217,6 @@ def make_llir(src, metadata, options, capability):
         kernels[0].set_calling_conv(amd.CALLING_CONV_AMDGPU_KERNEL)
         kernels[0].add_fn_attr("amdgpu-flat-work-group-size", f"1,{options.num_warps*options.warp_size}")
         kernels[0].add_fn_attr("amdgpu-waves-per-eu", f"{options.waves_per_eu}")
-        kernels[0].add_fn_attr("denormal-fp-math-f32", "preserve-sign")
```

Comment:
I'd suggest to set it to `ieee` explicitly. Later we want to expose it as a kernel arg.

Reply:
Done.